react/features/prejoin/utils.js

```diff
@@ -0,0 +1,802 @@
+// @flow
+
+export const countries = [
```

Comment:
Shouldn't this come from a config file?

@aaronkvanmeerten what do you think?

Reply:
What would be the benefit of coming from a config...? 🤔